playbooks/roles/bats/tasks/main.yml

```diff
@@ -22,6 +22,7 @@
     update: "{{ bats_update_forklift }}"
 
 - name: "Run bats"
-  shell: "bats {{ bats_tests }}"
+  shell: "bats {{ item }} 2>&1 | tee /root/{{ item }}.out"
```

Comment:
Fair question, `.out` was what it was before and Foremans systest were using. But I tend to agree that would make more sense.